server/controllers/AuthController.js

```diff
@@ -22,25 +23,35 @@ export default class AuthController {
    */
   static async createUser(request, response) {
     const {
-      firstName, lastName, email, password, avatarUrl
+      firstName, lastName, email, password, role, avatarUrl
     } = request.body;
+    let newUserFromAdmin;
     try {
+      if (!password) {
+        newUserFromAdmin = true;
+      }
       const existingUser = await User.findOne({ where: { email } });
       if (existingUser) return responseMessage(response, 409, { message: 'user already exist' });
       const newUser = await User.create({
-        firstName, lastName, email, password: bcrypt.hashSync(password, 10), avatarUrl
+        firstName, lastName, email, password: bcrypt.hashSync(password || 'password@1', 10), avatarUrl
```

Comment:
Lets make this default password a `process.env` key

Reply:
> Lets make this default password a `process.env` key

Done